troubleshooting.md

```diff
@@ -644,3 +644,39 @@ $ podman run --read-only --rootfs /path/to/rootfs ....
 
 Another option would be to create an overlay file system on the directory as a lower and then
 then allow podman to create the files on the upper.
+
+### 26) Running containers with CPU limits fails with a permissions error
+
+On some systems, non-root users do not have CPU limit delegation permissions.
```

Comment:
Could we be more specific here? We could say the issue happens with systemd as by default it doesn't enable all the available controllers for unprivileged users

Reply:
It's not *all* systems that have systemd. Fedora 33 does have CPU delegation enabled by default. It's really only systems that have cgroups v2 enabled but don't configure the delegations, which as far as I know is only Fedora 31 and 32. How specific should we be?

Reply:
the proposed solution works only on systems managed by systemd, that is why I was pointing out we could specify the issue is specific to systemd.  Either way, it is just a nit, I am fine as it is now.

Reply:
I see. That makes sense.